deployment/.env

```diff
@@ -0,0 +1,21 @@
+# database container
+# Database runs in a separate container, and container name is used as
+# its address. So we pass `postgress` instead of `localhost`.
+POSTGRES_HOST="postgres"
+POSTGRES_PORT=5432
+POSTGRES_USER="ednauser"
+POSTGRES_PASSWORD="ednapass"
+POSTGRES_DB="ednadb"
+
+# backend container
+EDNA_API_LISTEN_ADDR="*:9000"
+EDNA_API_SERVE_DOCS=true
+EDNA_DB_MAX_CONNECTIONS=200
+EDNA_DB_INITIALISATION_MODE="enable"
+# Init script is relative to backend image root
+EDNA_DB_INITIALISATION_INIT_SCRIPT="/init.sql"
+EDNA_DB_DEBUG=true
```

Comment:
Previously we didn't enable it, so we shouldn't enable it now as well. Application logging is sufficient to understand what's going on, dumping all DB operations is too verbose.

Reply:
I add it here because we already had this env in our readme
Do you propose to delete this variable at all?

Reply:
Not delete, can we do `EDNA_DB_DEBUG=false`?

Reply:
Oh, you are talking about default value, sure!